packages/runtime/runtime-definitions/src/dataStoreRegistry.ts

```diff
@@ -71,10 +71,17 @@ export interface IFluidDataStoreRegistry extends IProvideFluidDataStoreRegistry
 	 * @param name - The unique identifier of the data store registry entry to retrieve.
 	 * @returns A promise that resolves to the data store registry entry, or the entry itself, or undefined if not found.
 	 */
-	get(
-		name: string,
-	):
-		| Promise<FluidDataStoreRegistryEntry | undefined>
-		| FluidDataStoreRegistryEntry
-		| undefined;
+	get(name: string): Promise<FluidDataStoreRegistryEntry | undefined>;
```

Comment:
Comment is stale for `get`. Still says it might return the entry itself. Can fix after to get resolution faster.

Reply:
Nah - I was just misreading the intention of the comment.